### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by that-one-guy-alive**
Added A Request for the epic Mickey Rebrushed load remover to be integrated into live split

If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [Check ] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [Check ] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [Check ] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -23190,3 +23190,12 @@
         <Description>Autosplitter</Description>
     </AutoSplitter>
 </AutoSplitters>
+<Games>
+    <Game> Disney Epic Mickey: Rebrushed</Game>
+    <URLs>
+        <URL> https://raw.githubusercontent.com/that-one-guy-alive/Epic-Mickey-Load-Remover/refs/heads/main/rebrushedremover.asl</URL>
+    </URLs?
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:23197]:
```suggestion
    </URLs>
```

File: LiveSplit.AutoSplitters.xml [hunk 2]
```diff
@@ -23190,3 +23190,12 @@
...
+    <Type>Script</Type>
+    <Description> Load Removal Made Possible By NolifeVV</Description>
+    </AutoSplitter>
+</AutoSplitters>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:23201]: There shouldn't be more than 1 closing AutoSplitters tag. See how other PRs do it

File: LiveSplit.AutoSplitters.xml [hunk 3]
```diff
@@ -23190,3 +23190,12 @@
         <Description>Autosplitter</Description>
     </AutoSplitter>
 </AutoSplitters>
+<Games>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:23193]:
```suggestion
    <AutoSplitter>
        <Games>
```

File: LiveSplit.AutoSplitters.xml [hunk 4]
```diff
@@ -23189,4 +23189,13 @@
...
         <Description>Autosplitter</Description>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+    <Game> Disney Epic Mickey: Rebrushed</Game>
+    <URLs>
+        <URL> https://raw.githubusercontent.com/that-one-guy-alive/Epic-Mickey-Load-Remover/refs/heads/main/rebrushedremover.asl</URL>
+    </URLs>
+    <Type>Script</Type>
+    <Description> Load Removal Made Possible By NolifeVV</Description>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:23199]:
```suggestion
        <Game>Disney Epic Mickey: Rebrushed</Game>
    </Games>
    <URLs>
        <URL>https://raw.githubusercontent.com/that-one-guy-alive/Epic-Mickey-Load-Remover/refs/heads/main/rebrushedremover.asl</URL>
    </URLs>
    <Type>Script</Type>
    <Description>Load Removal Made Possible By NolifeVV</Description>
```

**Pull request merged**